### PR TITLE
gitsome: add page

### PR DESCRIPTION
**Pull request by waldyrious**
Note: PR created via the `hub` command :)

### CODE REVIEW

File: pages/common/gitsome.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# gitsome
+
+> A terminal-based interface for GitHub, accessed via the `gh` command.
+> It also provides menu-style autocomplete suggestions for git commands.
```
Comment (agnivade) [inline pages/common/gitsome.md]: Should we wrap "git" in backticks too ?

File: pages/common/gitsome.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`gh notifications`
+
+- List the current account's starred repos, filtered by a given search string:
+
+`gh starred {{python}}`
+
+- View the activity feed of a given GitHub repository:
+
+`gh feed tldr-pages/tldr -p`
```
Comment (agnivade) [inline pages/common/gitsome.md]: missing tokens :P

File: pages/common/gitsome.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- View the activity feed for a given GitHub user:
+
+`gh feed torvalds -p`
```
Comment (agnivade) [inline pages/common/gitsome.md]: same as above. Also can we expand on what `-p` is ?

File: pages/common/gitsome.md (hunk 1 above)
Comment (waldyrious) [inline pages/common/gitsome.md]: Yes, good catch.

File: pages/common/gitsome.md (hunk 2 above)
Comment (waldyrious) [inline pages/common/gitsome.md]: Gaah! Will fix :)

**Comment by agnivade** [comment]
Lovely PR.

**Comment by donnemartin** [comment]
@waldyrious looks great, thank you!

**Pull request merged**